mac_build/test_universal.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+
+folders_list="3rdParty/buildCache/mac/lib mac_build/build/Deployment"
+libs_list=$(find $folders_list -type f -name "*.a" -o -name "boinc")
+
+for lib in $libs_list; do
+    echo "Test lib: $lib"
+    if ! lipo $lib -verify_arch x86_64 arm64; then
```

Comment:
This `if` statement looks weird but according to my internal testing, it should work

Reply:
Charlie and you both verify it working.